### ISSUE TITLE
[a11y] Dismissing the remove home tile prompt is unintuitive for screen readers

### ISSUE THREAD

**Issue by mcomella** [mozilla-mobile/firefox-echo-show#65]
https://github.com/mozilla-mobile-skunkworks/firefox-connect/issues/461
